index.js

```diff
@@ -78,7 +78,7 @@ var validateMessage = function(raw) {
     }
 
     if (TYPES !== '*' && TYPES.indexOf(type) === -1) {
-      error('"%s" is not allowed type !', type);
+      error('"%s" is not allowed type ! Valid types are: %s', type, config.types.join(', '));
```

Comment:
Could we just use `TYPES` here rather than `config.types`? That way we wouldn't need to do the fancy double-assignment above right?